apps/src/code-studio/components/BounceCellEditor.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * @overview React component to allow for easy editing and creation of
+ * Bounce maps
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { SquareType } from '@cdo/apps/bounce/tiles';
+
+export default class BounceCellEditor extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event) {
+    const serializedArray = $(ReactDOM.findDOMNode(this)).serializeArray();
+    const values = serializedArray.reduce((prev, curr) => {
+      const value = isNaN(curr.value) ? undefined : Number(curr.value);
+      prev[curr.name] = value;
+      return prev;
+    }, {});
+    this.props.onUpdate(values);
```

Comment:
yeah, this is an unnecessary abstraction that will be addressed in https://github.com/code-dot-org/code-dot-org/pull/16080